upgradability/forward-proxy/forward_proxy/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "forward_proxy"
+version = "0.1.0"
+authors = ["Cardinals Cryptography"]
+edition = "2021"
+
+[dependencies]
+ink_primitives = { version = "3.4", default-features = false }
```

Comment:
hmm, maybe use the same version of ink libs in all the contracts, it's fine I guess to use 3 as major version